modules/advertlyBidAdapter.js

```diff
@@ -0,0 +1,129 @@
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { config } from '../src/config';
+import * as utils from '../src/utils';
+import {BANNER, VIDEO} from '../src/mediaTypes';
+import { ajax } from '../src/ajax';
+import {Renderer} from '../src/Renderer';
+
+const SUPPORTED_AD_TYPES = [BANNER, VIDEO];
+const BIDDER_CODE = 'advertly';
+const DOMAIN = 'http://api.advertly.com/';
+const RENDERER_URL = '//acdn.adnxs.com/video/outstream/ANOutstreamVideo.js';
+
+function isBidRequestValid(bid) {
+  return (typeof bid.params !== 'undefined' && parseInt(utils.getValue(bid.params, 'publisherId')) > 0);
+}
+
+function buildRequests(validBidRequests) {
+  return {
+    method: 'POST',
+    url: DOMAIN + 'www/admin/plugins/Prebid/getAd.php',
+    options: {
+      withCredentials: false,
+      crossOrigin: true
+    },
+    data: validBidRequests,
+  };
+}
+
+function interpretResponse(serverResponse, request) {
+  const response = serverResponse.body;
+  const bidResponses = [];
+  var bidRequestResponses = [];
+
+  utils._each(response, function(bidAd) {
+    bidAd.adResponse = {
+      content: bidAd.vastXml,
+      height: bidAd.height,
+      width: bidAd.width
+    };
+    bidAd.ttl = config.getConfig('_bidderTimeout')
+    bidAd.renderer = bidAd.context === 'outstream' ? createRenderer(bidAd, {
```

Comment:
Why is `bidAd.adUnitCode` included in all 3 parameters? Are all 3 parameters needed?

Reply:
Do you mean the 3 parameters are adResponse ,ttl, renderer ?

Reply:
I mean that all 3 parameters passed to `createRenderer` include `bidAd.adUnitCode`.
It seems like the only distinct data needed in that function is `bidAd` and `RENDERER_URL`

Reply:
Hi @msm0504 ,
Now we have removed bidAd.adUnitCode with in the second parameter in createRenderer function . Can you review this once?

Reply:
`bidAd.adUnitCode` is still being passed into `createRenderer ` twice.

Why is `bidAd.adUnitCode` used on line 74 and `adUnitCode` used on line 78? Can't both use `bidAd.adUnitCode`, which would eliminate the need for a 3rd param in that function?

Reply:
if we use bidAd.adUnitCode for  line 78, we have issue on ci/circleci

Reply:
That line needs to specify the property name.

`adUnitCode: bidAd.adUnitCode`

Reply:
adUnitCode updated